dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -1,41 +1,55 @@
 -# English only for now
 - if language == "en"
```

Comment:
In theory it can appear on any page, but at the moment it just appears on studio.code.org homepage.